No Auth Provider found for name "gcp"

<!-- This form is for bug reports and feature requests ONLY!

If you're looking for a help then check our [Slack Channel](http://volcano-sh.slack.com/) or have a look at our [dev mailing](https://groups.google.com/forum/#!forum/volcano-sh) 
-->

**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
> /kind bug

**What happened**:
panic: No Auth Provider found for name "gcp"

```
goroutine 1 [running]:
volcano.sh/volcano/vendor/github.com/kubernetes-sigs/kube-batch/pkg/client/clientset/versioned/typed/scheduling/v1alpha1.NewForConfigOrDie(...)
        /home/miguelangel_prieto_martos/gopath/src/volcano.sh/volcano/vendor/github.com/kubernetes-sigs/kube-batch/pkg/client/clientset/versioned/typed/scheduling/v1alpha1/scheduling_client.go:65
volcano.sh/volcano/vendor/github.com/kubernetes-sigs/kube-batch/pkg/client/clientset/versioned.NewForConfigOrDie(0xc000288000, 0x0)
        /home/miguelangel_prieto_martos/gopath/src/volcano.sh/volcano/vendor/github.com/kubernetes-sigs/kube-batch/pkg/client/clientset/versioned/clientset.go:85 +0x10d
volcano.sh/volcano/pkg/cli/queue.ListQueue(0xc0000bfd08, 0x0)
        /home/miguelangel_prieto_martos/gopath/src/volcano.sh/volcano/pkg/cli/queue/list.go:57 +0x7f
main.buildQueueCmd.func2(0xc000204f00, 0x1c095b0, 0x0, 0x0)
        /home/miguelangel_prieto_martos/gopath/src/volcano.sh/volcano/cmd/cli/queue.go:45 +0x22
volcano.sh/volcano/vendor/github.com/spf13/cobra.(*Command).execute(0xc000204f00, 0x1c095b0, 0x0, 0x0, 0xc000204f00, 0x1c095b0)
        /home/miguelangel_prieto_martos/gopath/src/volcano.sh/volcano/vendor/github.com/spf13/cobra/command.go:766 +0x2ae
volcano.sh/volcano/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc000107900, 0xc0000bff70, 0x1, 0x1)
        /home/miguelangel_prieto_martos/gopath/src/volcano.sh/volcano/vendor/github.com/spf13/cobra/command.go:852 +0x2c0
volcano.sh/volcano/vendor/github.com/spf13/cobra.(*Command).Execute(...)
        /home/miguelangel_prieto_martos/gopath/src/volcano.sh/volcano/vendor/github.com/spf13/cobra/command.go:800
main.main()
        /home/miguelangel_prieto_martos/gopath/src/volcano.sh/volcano/cmd/cli/vkctl.go:47 +0x194
```

**What you expected to happen**:
No error

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:
I have resolved this issue with the import _ "k8s.io/client-go/plugin/pkg/client/auth/gcp" in volcano.sh/volcano/pkg/cli/queue/util.go

**Environment**:
- Volcano Version: release-0.1
- Kubernetes version (use `kubectl version`): v1.13.5-gke.10
- Cloud provider or hardware configuration: GCP
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

--- a/pkg/cli/queue/util.go
+++ b/pkg/cli/queue/util.go
@@ -21,6 +21,8 @@ import (
 
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
+	// Initialize client auth plugin.
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 )
 
 func homeDir() string {

